core/html_api.php

```diff
@@ -642,7 +642,8 @@ function print_manage_menu( $p_page = '' ) {
 
 	# Plugins menu items - these are cooked links
 	foreach( $t_menu_options as $t_menu_item ) {
-		echo '<li>', $t_menu_item, '</li>';
+		$t_active = $p_page && strpos( $t_menu_item, $p_page ) !== false ? 'class="active"' : '';
+		echo "<li $t_active>", $t_menu_item, '</li>';
```

Comment:
I would have the space as part of the $t_active string, so it if it is not set, then you end up with no space.